### DISCUSSION TITLE
API query parameter problem

### DISCUSSION THREAD

**Discussion by galatea99** [pocketbase/pocketbase#2515] [Q&A] [answered]
Does anybody have the same issue like when doing: 
http://localhost:8090/api/collections/events/records/?filter=(event_date>'2023-05-23'  &&  status='1')
will get an error:
{
    "code": 400,
    "message": "Invalid filter parameters.",
    "data": {}
}
however, http://localhost:8090/api/collections/events/records/?filter=(event_date>'2023-05-23' )
runs fine.
Anything you added to the filter, will cause an error. 
It's not an quote (') problem as I have tried all sorts of combination.

Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that the filter value is properly url encoded (_this is done automatically if you use the JS or Dart SDKs_). For example the encoded version of the first url is:

```
http://localhost:8090/api/collections/events/records/?filter=%28event_date%3E%272023-05-23%27%20%26%26%20status%3D%271%27%29
```

**Reply by galatea99** [reply]
Thanks for such prompt response! 
It works now
